lms/djangoapps/teams/static/teams/templates/teams_tab.underscore

```diff
@@ -7,6 +7,7 @@
     </div>
 </div>
 <div class="teams-header"></div>
+<div class="teams-instructor-tools-bar"></div>
```

Comment:
Being this high up in the `teams_tab` html hierarchy, should I make this more general than `instructor-tools-bar`? Will there ever be anything else that might fit into this "outside of both `teams-header` and `teams-main`" pattern?